src/gsi/setuppm2_5.f90

```diff
@@ -770,21 +779,30 @@ subroutine setuppm2_5(obsLL,odiagLL,lunin,mype,nreal,nobs,isis,is,conv_diagsave)
                 mype,nfldsig)
           call tintrp2a11(pm25wc(:,:,:,2,nfldsig),pm25wc_ges(2),dlat,dlon,dtime,hrdifsig,&
                 mype,nfldsig)
-
-          if (pm25wc_ges(1) >= 1.0_r_kind) then
+          !write(6,"(A,L3,F8.3,1x,F8.3,F6.2,4F7.2)")"OBS_INF ",muse(i),data(ilate,i),data(ilone,i),veg_type_ges,pm2_5ges,innov,pm25wc_ges(1),pm25wc_ges(2)
+          if (pm25wc_ges(1) >= 2.0_r_kind) then
             pm25wc_ges(1)=1.0_r_kind
           else
-            pm25wc_ges(2)=0.0_r_kind
+            pm25wc_ges(1)=0.0_r_kind
           end if
-          if (pm25wc_ges(2) >= 1.0_r_kind) then
+          if (pm25wc_ges(2) >= 2.0_r_kind) then
```

Comment:
Is the change from 1.0 to 2.0 a bug fix or a threshold change? For the latter, will it be better to use a parameter to control this?